# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: dom/indexedDB/SerializationHelpers.h
# Commit: 94ca5d86b3b0
# Full Hash: 94ca5d86b3b0c398294dd82d44194a5173c5aec6
# Author: Simon Giesecke <sgiesecke@mozilla.com>
# Date: 2020-01-09 21:39:42
# Regressor Bug: 1600906
# File Overlap Count: 1
# Description:
#   Bug 1600906 - Use scoped enums in IDBCursor. r=dom-workers-and-storage-reviewers,ytausky
#   
#   Depends on D57990
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D57991
# ==============================================================================

diff -r 801f97f67366 -r 94ca5d86b3b0 dom/indexedDB/SerializationHelpers.h
--- a/dom/indexedDB/SerializationHelpers.h	Thu Jan 09 14:58:17 2020 +0000
+++ b/dom/indexedDB/SerializationHelpers.h	Mon Jan 06 14:19:39 2020 +0000
@@ -71,8 +71,9 @@
 template <>
 struct ParamTraits<mozilla::dom::IDBCursor::Direction>
     : public ContiguousEnumSerializer<
-          mozilla::dom::IDBCursor::Direction, mozilla::dom::IDBCursor::NEXT,
-          mozilla::dom::IDBCursor::DIRECTION_INVALID> {};
+          mozilla::dom::IDBCursor::Direction,
+          mozilla::dom::IDBCursor::Direction::Next,
+          mozilla::dom::IDBCursor::Direction::Invalid> {};
 
 template <>
 struct ParamTraits<mozilla::dom::IDBTransaction::Mode>
